guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+
+### Concept topics
+
+Answer "what-is" questions and provide essential background information that users must know.
+You'll find most concept topics in the Overview section, but they can be useful elsewhere too.
+
+Use nominal style for the title, for example, "Security" or "Security Concept".
+
+*Not sure we need much predefined structure here; it may depend on the content.*
+
+> [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)
+> Conceptual information might explain the nature and components of a product and describe how it fits into a category of products. Conceptual information helps readers to map their knowledge and understanding to the tasks they need to perform and to provide other essential information about a product, process, or system.
+
+You can use the [concept topic template](../templates/resources/concept.md).
+
+### Task (tutorial) topics
+
+Provide "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure.
+
+Select a title that describes the task that's accomplished, not the documented software feature. For example, use "Define resource consumption", not "Select a profile". You can use the gerund form "Selecting...", imperative "Select...", or "How to select...".
+
+With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
+It's good practice to have 5-9 steps; anything longer can probably be split.
+
+*For longer code blocks, I'd like to use an expandable section as described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) - but not sure whether our website supports this.*
+
+> [- Dita Task Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-task-topic.html)
+> A task information type answers the "How do I?" question by providing precise step-by-step instructions detailing the requirements that must be fulfilled, the actions that must be performed, and the order in which the actions must be performed. The task topic includes sections for describing the context, prerequisites, expected results, and other aspects of a task.
+
+You can use the [task topic template](../templates/resources/task.md).
+
+### Reference topics
+
+Typically organized into one or more sections containing a list or table with data that is usually “looked up” rather than memorized.
+
+Use nominal style for the title, for example, "Configuration Parameters".
```

Comment:
When I read "Switching registries at runtime", I expect to find an instruction how to switch registries at runtime, so, a task topic with a number of steps. Looking at the content of this one, I believe it implicitly does instruct users how to enable switching registries. So it might be better suited under "Operations" maybe?